tools/certissuer/storage_test.go

```
@@ -0,0 +1,150 @@
+	require.Len(t, archive, len(domainFiles))
+	assert.Regexp(t, `\d+\.`+regexp.QuoteMeta(domain), archive[0].Name())
+}
+
+func generateTestFiles(t *testing.T, dir, domain string) []string {
+	t.Helper()
+
+	var filenames []string
+
+	for _, ext := range []string{extIssuer, extCert, extKey, extPem, extPfx, extResource} {
+		filename := filepath.Join(dir, domain+ext)
+		err := os.WriteFile(filename, []byte("test"), 0o666)
```

golangci-lint (1.64.8)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)